Bug: When deployed to heroku *.css files return 500 responses

Trying to deploy a prototype to heroku and I'm finding that the `application.css` and `example.css` return 500 responses with the message:

```
Error: Cannot find module 'css' at Function.Module._resolveFilename (module.js:338:15) at Function.Module._load (module.js:280:25) at Module.require (module.js:364:17) at require (module.js:380:17) at new View (/app/node_modules/express/lib/view.js:43:49) at Function.app.render (/app/node_modules/express/lib/application.js:486:12) at ServerResponse.res.render (/app/node_modules/express/lib/response.js:798:7) at /app/server.js:28:6 at callbacks (/app/node_modules/express/lib/router/index.js:164:37) at param (/app/node_modules/express/lib/router/index.js:138:11)
```

I've cloned this repo fresh and pushed it straight to heroku without any changes at http://powerful-coast-8011.herokuapp.com/. You can see the error on it's [application.css](http://powerful-coast-8011.herokuapp.com/public/stylesheets/application.css).

Seems to me that the express static middleware isn't handling calls to /public and instead it's trying to render the file using the auto-render block:

```
// auto render any view that exists

app.get(/^\/(.+)/, function (req, res) {

    var path = (req.params[0]);

    res.render(path, {'assetPath' : assetPath }, function(err, html) {
        if (err) {
            console.log(err);
            res.send(404);
        } else {
            res.end(html);
        }
    });

});
```

The full response from heroku when deploying is this gist  https://gist.github.com/michaeldfallen/d3aaa6f97e8aab47cdd0

--- a/Procfile
+++ b/Procfile
@@ -1,1 +1,1 @@
-web: node server.js
+web: node start.js

